spec/support/helpers/rack_helpers_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+RSpec.describe RackHelpers, type: :helper do
+  describe '#create_helper' do
+    subject(:helper) { create_rack_request(env) }
+
+    context 'without args' do
+      let(:env) { {} }
+
+      it 'returns Rack::Request instance double with default values' do
+        expect(helper).to be_an_instance_of(RSpec::Mocks::InstanceVerifyingDouble)
+        expect(helper.path).to eq('/')
+        expect(helper.params).to eq({})
+        expect(helper.env).to eq({})
+      end
+    end
+
+    context 'with args' do
+      let(:env) { { path: '/42', params: { a: 1 }, env: { b: 2 } } }
```

Comment:
may be this should have another name. env[:env] looks strange

Reply:
Fixed.